src/building/prerequisites.md

```diff
@@ -0,0 +1,29 @@
+# Prerequisites
+
+Before building the compiler, you need the following things installed:
```

Comment:
https://github.com/rust-lang/rust/blob/master/README.md lists the minimum versions required and I think it would nice to have them here as well.